Source/Scene/Expression.js

```diff
@@ -1298,6 +1318,11 @@ define([
                 if (value === 'TILES3D_TILESET_TIME') {
                     return 'u_tilesetTime';
                 }
+                break;
+            case ExpressionNodeType.TERNARY:
+                if (defined(ternaryFunctions[value])) {
+                    return value + '(' + left + ', ' + right + ', ' + test + ')';
+                }
```

Comment:
@lilleyse isExactClass, isClass, getExactClassName are functions that are also now failing in my code. The only other  place I see this referernced is in [PR #4625](https://github.com/AnalyticalGraphicsInc/cesium/pull/4625)